### PR TITLE
Test purging of ipro-rewritten resources.

### PR DESCRIPTION
**Pull request by jmarantz**
adds test for https://github.com/pagespeed/ngx_pagespeed/issues/1192

See https://github.com/pagespeed/ngx_pagespeed/pull/1193 for a fix.

### CODE REVIEW

File: pagespeed/system/system_test.sh [hunk 1]
```diff
@@ -2143,6 +2143,7 @@ function cache_purge_test() {
       echo $CURL --request PURGE --proxy $SECONDARY_HOSTNAME "$PURGE_URL"
       check $CURL --request PURGE --proxy $SECONDARY_HOSTNAME "$PURGE_URL"
     fi
+    echo ""
```
Comment (oschaaf) [inline pagespeed/system/system_test.sh:2146]: Left in by accident?

**Comment by oschaaf** [comment]
travis complaints seem unrelated (font api css inlining failure)
had one remark about what seems to be an unintended echo line, otherwise LGTM

File: pagespeed/system/system_test.sh (hunk 1 above)
Comment (jmarantz) [inline pagespeed/system/system_test.sh:2146]: actually, no, this was on purpose.  For years I've been bothered a little bit by this message in the system-test logs:
    Purge successfulsleep 6
this brilliant bit of scripting makes it
    Purge successful
    sleep 6

Comment (oschaaf) [inline pagespeed/system/system_test.sh:2146]: Ah cool :-)

**Comment by jmarantz** [comment]
fyi this test implementation needs to be cleaned up; it writes files into a source-controlled area and this causes problems in our test scripts for a few reasons.

**Comment by jmarantz** [comment]
Closing this pull-request; I need to push this one out in the other direction.  This business of writing files in a source-controlled area was something we were doing before, but it causes races in our test environment that I need a more infrastructural fix for.

**Pull request closed without merging**